extras/flux/patch-gitrepository-config-private.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: GitRepository
+metadata:
+  name: giantswarm-config
+  namespace: flux-giantswarm
+spec:
+  include:
+    - repository:
+        name: shared-configs
+      fromPath: default
+      toPath: default
+    - repository:
+        name: shared-configs
+      fromPath: include
+      toPath: include
+  secretRef:
+    name: configs-ssh-credentials
+  url: ssh://git@ssh.github.com:8081/giantswarm/${customer_codename}-configs.git
```

Comment:
Why is the port hard-coded whilst the port in this is not? https://github.com/giantswarm/management-cluster-bases/pull/72/files#diff-cf9b61ade9ecc3b06ceef4eed48affe02b75acde612239972814ec422217a636

Reply:
It is experimental yet. I just hardcoded the other one too as generally we use 443 and provide patches for private clusters.